recipes/glad/all/conandata.yml

```diff
@@ -1,8 +1,14 @@
 sources:
+  "0.1.34":
+    url: "https://github.com/Dav1dde/glad/archive/v0.1.34.tar.gz"
+    sha256: "4be2900ff76ac71a2aab7a8be301eb4c0338491c7e205693435b09aad4969ecd"
   "0.1.33":
-    url: "https://github.com/Dav1dde/glad/archive/v0.1.33.zip"
-    sha256: "3175b56aa5ff16c5bfb33df3f36d6a7eb680e7d9f1675035472015e007a88660"
+    url: "https://github.com/Dav1dde/glad/archive/v0.1.33.tar.gz"
+    sha256: "4bc850062e9bcfc182bd4095d45b987517507d8ec0edcc424ed58f67b4ea9e37"
 patches:
+  "0.1.34":
+    - patch_file: "patches/0002-CMake-handle-all-specs.patch"
```

Comment:
Should we version the patches (by e.g. prepending the version to the patches)?
e.g. `0.1.34-0001XXX.patch` and `0.1.35-0001XXX.patch`

Reply:
yes maybe, let's do this in the next version (next year? ;) )